pages/history.tsx

```diff
@@ -103,8 +125,25 @@ const History = () => {
         ),
       ]
         .concat(
-          transfers.map(
-            x => [-1, { ui: { timestamp: x.timestamp }, ...x }] as any
+          transfers[0].map(
+            x =>
+              [
+                TransferType.MUTEZ,
+                { ui: { timestamp: x.timestamp }, ...x },
+              ] as any
+          )
+        )
+        .concat(
+          transfers[1].map(
+            x =>
+              [
+                x.token.standard === "fa2"
+                  ? TransferType.FA2
+                  : x.token.standard === "fa1.2"
+                  ? TransferType.FA1_2
+                  : TransferType.UNKNOWN,
```

Comment:
I didn't read the rest of the code (TL;DR, sorry sorry) but how do you decide the standard of a token in TzSafe?

Reply:
It's provided by tzkt api. which actually led to some weird bug because tzkt was identifying some contract being fa2 when it's wasn't compliant with the spec